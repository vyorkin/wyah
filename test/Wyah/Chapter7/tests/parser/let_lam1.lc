let f = \y -> y - x;
