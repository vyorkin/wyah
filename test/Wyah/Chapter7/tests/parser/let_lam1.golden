Program [Decl "f" (ELam (Var "y") (EOp Sub (EVar (Var "y")) (EVar (Var "x"))))]